src/models.js

```diff
@@ -16,7 +16,7 @@ export class TxHistory {
    *   timestamp: number;
    *   tokenUid: string;
    *   balance: number;
-   *   voided: boolean;
+   *   isVoided: boolean;
```

Comment:
question: Is this correct?

Reply:
Yes, it is, it is 🤙. The document that was wrong.